publish-common/src/main/java/com/ericsson/eiffel/remrem/publish/service/GenerateURLTemplate.java

```diff
@@ -49,7 +49,7 @@ public void setGenerateServerPath(String generateServerPath) {
     }
 
     public String getUrl() {
-        return generateServerUri + generateServerPath + "/{mp}?msgType={msgType}";
+        return generateServerUri + (generateServerPath == null ? "" : generateServerPath) + "/{mp}?msgType={msgType}";
```

Comment:
If we provide a slash in the end for generateServerPath it will result in error?
can you just validate if the generateServerPath ends with a slash and remove it before adding the "/{mp}?msgType={msgType}";

Reply:
If we provide a slash in the generateServerPath, it will work fine. I have tested in my local.